ios/brave-ios/Sources/Brave/Frontend/Browser/Toolbars/PageSecurityView.swift

```diff
@@ -63,9 +64,15 @@ struct PageSecurityView: View {
   var body: some View {
     VStack(alignment: .leading, spacing: 0) {
       VStack(alignment: .leading, spacing: 16) {
-        Text(displayURL)
+        URLElidedText(text: displayURL)
           .font(.headline)
           .foregroundStyle(Color(braveSystemName: .textPrimary))
+          .truncationMode(.tail)
+          .frame(maxWidth: .infinity, alignment: .leading)
+          .fixedSize(horizontal: false, vertical: true)
+          .environment(\.layoutDirection, .leftToRight)
+          .flipsForRightToLeftLayoutDirection(false)
```

Comment:
These are better put in `URLElidedText` directly since you'd probably want it anywhere its used

Reply:
Fixed.